packages/redux/src/entities/types/productListing.types.ts

```diff
@@ -7,11 +7,12 @@ import type {
 } from '@farfetch/blackout-client';
 import type { ProductEntity } from './product.types.js';
 
-export type FacetGroupsNormalized = Array<
-  Omit<FacetGroup, 'values'> & {
-    values: FacetEntity['id'][][];
-  }
->;
+export type FacetGroupNormalized = Omit<FacetGroup, 'values'> & {
+  values: FacetEntity['id'][][];
+  hash?: string;
```

Comment:
I think this should be removed since it should not be consumed by clients, even though that property exists. Did you need to use it in boilerplate?

Reply:
I thought it was needed, but a better approach was taken from the boilerplate side! Removed.